src/tcgwars/logic/impl/gen4/CallOfLegends.groovy

```diff
@@ -804,343 +465,110 @@ public enum CallOfLegends implements LogicCardInfo {
           weakness R
           resistance W, MINUS20
           move "Grind", {
```

Comment:
This is one of the ones that it would be nice to have an attackRequirement on as well.